mitmproxy/utils/typecheck.py

```diff
@@ -26,7 +26,7 @@ def check_type(attr_name: str, value: typing.Any, typeinfo: type) -> None:
     if typename.startswith("typing.Union"):
         try:
             types = typeinfo.__args__
-        except AttributeError:
+        except AttributeError:  # pragma: no cover
```

Comment:
Oh yes you are right - the pattern just looks too similar to the import clauses we used to have. I will mock it to test it properly.